Cargo.toml

```diff
@@ -11,17 +11,18 @@ categories = ["embedded", "no-std", "network-programming"]
 readme = "README.md"
 
 [dependencies]
-sha1 = "0.6"
-heapless = "0.5"
-byteorder = { version = "1.4", default-features = false }
-httparse = { version = "1.4", default-features = false }
-rand_core = "0.6"
+sha1 = "0.10.1"
+heapless = "0.7.14"
+byteorder = { version = "1.4.3", default-features = false }
+httparse = { version = "1.7.1", default-features = false }
+rand_core = "0.6.3"
```

Comment:
Same with sha1 above

Reply:
I check it with `cargo feature`. All of them should be no_std by default.